platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -31,11 +31,23 @@ MEMORY
 #if defined(S_CODE_SRAM_ALIAS_BASE)
   CODE_RAM (rwx) : ORIGIN = S_CODE_SRAM_ALIAS_BASE, LENGTH = TOTAL_CODE_SRAM_SIZE
 #endif
-#ifndef TFM_MULTI_CORE_TOPOLOGY
+#if !defined(TFM_MULTI_CORE_TOPOLOGY) && !defined(CMSE_VENEER_REGION_IN_CODE)
   VENEERS  (rx)  : ORIGIN = CMSE_VENEER_REGION_START, LENGTH = CMSE_VENEER_REGION_SIZE
 #endif
 }
 
+/* Define this macro in the platform's region_defs.h to override the default
```

Comment:
Can you put this into #ifdef CMSE_VENEER_REGION_IN_CODE?
I propose to decouple completely the solution when VENEERS are in code and when they are not. This is much easier to read, and it is not too much of duplication..

Reply:
Can you please elaborate? I don't know exactly what you are asking for.

I suppose without _IN_CODE there's less of a reason for alignment since platforms fully control the placement, so adding an ifdef is fine with me.

I really don't want to duplicate the sgstubs stuff though if that's what you meant since it's so finicky, and if they go out of sync it will be a lot of work to understand any changes and apply them after the fact.

Reply:
I changed this somewhat, making it less verbose, take a look.

Reply:
Good enough for now, I am not 100% convinced, but I don't feel i should delay this more, for now